scout/dashboard.md

```diff
@@ -41,12 +41,33 @@ The repository tag list shows all tags for a repository. You can search for spec
 
 Each entry in the list shows the following details:
 
+- A checkbox to mark the tag for comparison to one other.
+
+  > **Tip**
+  >
+  > Compare two tag images by selecting the checkboxes next to them and selecting the **Compare images** button at the top of the list.
```

Comment:
I don't think "tag image" is a correct term. 
I suggest: "two image tags", "two tags", "two tagged images" or just "two images"?